benchmark/run_trex.sh

```diff
@@ -9,16 +9,39 @@
 # ----------------------------------------------------------------------------
 
 # usage: run T-REX software
-gene_tree_dir=$1
-output_fp=$2
-verbose=$3
-stdout=$4
-stderr=$5
-scripts_dir=$6
-species_tree_fp=$7
-input_file_nwk=$8
-trex_install_dir=$9
-base_input_file_nwk=${10}
+
+# -e: script will exit if any command fails
+# -u: force initialization of all variables
+set -eu
+
+# define parameters using an array
+args=(
+  gene_tree_dir  # hey
+  output_fp      # look
+  verbose        # I can comment each parameter
```

Comment:
Great solution @qiyunzhu !

Could you update the comments and I'll merge. Thanks!